### PR TITLE
[releng] Mention kubernetes/websites everywhere in the same way

### PR DESCRIPTION
**Pull request by MaxymVlasov**
Related to https://kubernetes.slack.com/archives/C0191RDKHU1/p1680549820387879

### CODE REVIEW

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to me.

/lgtm

File: scripts/releng/update-release-info.sh
Comment (sftim) [inline scripts/releng/update-release-info.sh:1]: THIS CONTENT IS AUTO-GENERATED via `./scripts/releng/update-release-info.sh` in [kubernetes/website](https://github.com/kubernetes/website)

I think we should update the script.

File: scripts/releng/update-release-info.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ title: Kubernetes Release Cycle
 type: docs
 auto_generated: true
 ---
-<!-- THIS CONTENT IS AUTO-GENERATED via ./scripts/releng/update-release-info.sh in k/website -->
+<!-- THIS CONTENT IS AUTO-GENERATED via ./scripts/releng/update-release-info.sh in kubernetes/website -->
```
Comment (sftim) [inline scripts/releng/update-release-info.sh:11]: I prefer either of:
- `k/website`
- https://github.com/kubernetes/website

`kubernetes/website` doesn't provide enough context for the uninitated, and is more typing for those in the know.

File: scripts/releng/update-release-info.sh
Comment (MaxymVlasov) [reply scripts/releng/update-release-info.sh:1]: Make sense. Checkout new proposal

**Review by Arhell — COMMENTED** [review]
/lgtm

**Comment by divya-mohan0209** [comment]
@kubernetes/release-engineering : PTAL

**Comment by palnabarun** [comment]
QQ, Does https://github.com/kubernetes/website/pull/40493 supersede this PR?

**Comment by divya-mohan0209** [comment]
Nope, afaik, it doesn't supersede the current PR. This PR was [suggested](https://github.com/kubernetes/website/pull/40493/files#r1257654553) as a branch-out since the script was required to be amended. The explicit changes to release.md will be discarded on PR # 40493.

**Review by saschagrunert — APPROVED** [review]

**Review by palnabarun — APPROVED** [review]
/lgtm
/approve

**Review by xmudrii — APPROVED** [review]
/lgtm
/approve
@MaxymVlasov This PR might require force-pushing again as the Netlify build broke. I don't think we have easy way to restart it.

**Comment by MaxymVlasov** [comment]
@xmudrii done

**Review by xmudrii — APPROVED** [review]
/lgtm
/approve

**Pull request merged**